barebox/env/init/ps1

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ ${global.allow_color} = "1" ]; then
```

Comment:
The prompt doesn't have color by default with this configuration. Do we want to set `global.allow_color` somewhere?

Reply:
It's set here: https://github.com/grisp/grisp2-rtems-toolchain/pull/25/files#diff-2fa831a9501463be1c122c52104851b0d93e708323e08dd500918cadec2f359fR235